tests/extensions/test_file.py

```diff
@@ -9,8 +9,8 @@
 
 class ByteOrderTest(unittest.TestCase):
     def test_to_str(self) -> None:
-        self.assertEqual(str(ByteOrder.LITTLE_ENDIAN), "little-endian")
-        self.assertEqual(str(ByteOrder.BIG_ENDIAN), "big-endian")
+        self.assertEqual(ByteOrder.LITTLE_ENDIAN.value, "little-endian")
+        self.assertEqual(ByteOrder.BIG_ENDIAN.value, "big-endian")
```

Comment:
I think the only purpose for these tests was to provide test coverage for the `__str__` methods. Does coverage drop if we exclude these? It doesn't seem that useful to me to just test an Enum's value.

Reply:
According to the report above the coverage actually went *up* after this change :)

Reply:
Sorry, should have been more clear in the comment. I realize the current changes have increased coverage. I'm wondering if we can remove these tests entirely. It's probably not crucial for this PR, but something to consider since they don't seem to add much value.